lib/DDG/Spice/SublimePackages.pm

```diff
@@ -24,8 +23,16 @@ my $skip = join "|", share('skipwords.txt')->slurp(chomp => 1);
 
 handle remainder => sub {
     return unless $_; 
+    # OS search filters
+    s/linux/:linux/g; 
+    s/windows/:win/g;
+    s/mac\s?os\s?x|os\s?x/:osx/g;
+    # Version search filters
+    s/version 2/:st2/g;
+    s/version 3/:st3/g;
```

Comment:
@MrChrisW this seems fine -- maybe s/v?(ersion)?2/.../` ? e.g. "sublime text 2", "sublime text v3"

This may be premature optimization, but I think those are reasonable for now.